airbyte-protocol/models/src/main/resources/airbyte_protocol/airbyte_message.yaml

```diff
@@ -118,10 +118,43 @@ definitions:
       name:
         type: string
         description: Stream's name.
-      json_schema:
-        description: Stream schema using Json Schema specs.
+      discovered:
```

Comment:
maybe `schema` or `schema_info` or something? can't think of a great name but the current name is super non-descriptive

Reply:
moot with new structure.